README.md

```diff
@@ -1,4 +1,4 @@
-# mdi-react [![npm package](https://img.shields.io/npm/v/mdi-react.svg?style=flat-square)](https://npmjs.org/package/mdi-react) [![Material Design Icons version](https://img.shields.io/badge/mdi-v2.3.54-blue.svg?style=flat-square)](https://materialdesignicons.com)
+# mdi-react [![npm package](https://img.shields.io/npm/v/mdi-react.svg?style=flat-square)](https://npmjs.org/package/mdi-react) [![Material Design Icons version](https://img.shields.io/badge/dynamic/json.svg?label=mdi&url=https%3A%2F%2Fraw.githubusercontent.com%2Flevrik%2Fmdi-react%2Fmaster%2Fpackage.json&prefix=v&query=devDependencies%5B%22%40mdi%2Fsvg%22%5D&colorB=%23007ec6&style=flat-square)](https://materialdesignicons.com) [![devDependencies Status](https://david-dm.org/levrik/mdi-react/dev-status.svg?style=flat-square)](https://david-dm.org/levrik/mdi-react?type=dev)
```

Comment:
I don't like the devDependencies status. There are a lot of other dependencies inside it a user doesn't care about since they are only used for building the final package. If you can make it working that it only monitors the `@svg/mdi` dependency I would be fine.

Reply:
Hm, I could put only `@mdi/svg` into `dependencies`

Reply:
The only other type of deps users will are about is `peerDependencies` for `react`.